app/src/main/java/com/sharesmile/share/rfac/fragments/ShareFragment.java

```diff
@@ -129,13 +134,17 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         Bundle arg = getArguments();
         mCauseData = (CauseData) arg.getSerializable(BUNDLE_CAUSE_DATA);
-        mWorkoutData = (WorkoutData) arg.getParcelable(WORKOUT_DATA);
+        mWorkoutData = arg.getParcelable(WORKOUT_DATA);
         mShowLogin = arg.getBoolean(BUNDLE_SHOW_LOGIN);
     }
 
     @Nullable
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+
+        RadioButton button = (RadioButton)mShareGroup.findViewById(R.id.fb);
+        button.setButtonDrawable(VectorDrawableCompat.create(getResources(), R.drawable.selector_fb_radio, null));

```

Comment:
This is not required now, we have moved to app compat library 23.4.0, and using 
static
    {
        AppCompatDelegate.setCompatVectorFromResourcesEnabled(true);
    }
code block in all activities.
This fixes the vectorDrawable crash.